include/eve/module/core/regular/impl/simd/riscv/if_else.hpp

```diff
@@ -0,0 +1,63 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/function/simd/riscv/make.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
```

Comment:
I see that in other files here wide and logical are passed by reference, but that's a mistake - they are always passed by value. 

Also - you don't need to put noexcepts. It doesn't matter one way or the other but i generally don't.

Reply:
> I see that in other files here wide and logical are passed by reference, but that's a mistake - they are always passed by value.

Fixed. By the way, it seems in arm SVE you use const refs:

https://github.com/jfalcou/eve/blob/main/include/eve/detail/function/simd/arm/sve/combine.hpp#L19

> you don't need to put noexcepts

Well, AFAIK for `noexcept` functions compiler could be more aggressive with optimizations. Here it should not be the real point though, but I prefer for now to keep them, if you are not against.

Reply:
noexcept only make code better when used in place where noexcept quality of function is important : move semantic, copy etc. Mostly in standard container. We do mark them noexcept because they are and mayeb , one day, maybe, it'll have an impact. But the code itself of the function don't really benefit from this.

Reply:
noexcept is completly irrelevant for force inline funtions. w/e.